x-pack/plugins/canvas/canvas_plugin_src/renderers/markdown/index.tsx

```diff
@@ -29,24 +29,20 @@ export const getMarkdownRenderer =
     render(domNode, config, handlers) {
       const fontStyle = config.font ? config.font.spec : {};
 
-      if (config.content) {
-        ReactDOM.render(
-          <KibanaThemeProvider theme={{ theme$ }}>
-            <Markdown
-              className="canvasMarkdown"
-              style={fontStyle as CSSProperties}
-              openLinksInNewTab={config.openLinksInNewTab}
-              readOnly
-            >
-              {config.content}
-            </Markdown>
-          </KibanaThemeProvider>,
-          domNode,
-          () => handlers.done()
-        );
+      ReactDOM.render(
```

Comment:
it's but doesn't quite look like it because there was an intermediate PR to address an issue with the canvas markdown rendering [here](https://github.com/elastic/kibana/pull/177989)

Reply:
Ah yes, I remember this. Thank you for the clarification!